apps/src/turtle/turtle.js

```diff
@@ -727,18 +711,9 @@ Artist.prototype.drawImages = function () {
  */
 Artist.prototype.loadTurtle = function (initializing = true) {
   const onloadCallback = initializing ? this.display : this.drawTurtle;
-  this.avatarImage.onload = _.bind(onloadCallback, this);
+  this.avatar.image.onload = _.bind(onloadCallback, this);
```

Comment:
you'll need to declare the `new Image()` somewhere. I vote here.

Reply:
Right, of course :facepalm: